public/js/clients/chrome.js

```diff
@@ -138,6 +138,19 @@ function connectClient() {
   return deferred.promise;
 }
 
+function connectWebsocket(websocketUrl) {
+  return new Promise(resolve => {
+    bootstrap(InspectorBackend);
+    WebSocketConnection.Create(
+      websocketUrl,
+      conn => {
+        connection = conn;
+        resolve();
+      }
+    );
+  });
+}
```

Comment:
nice - i'm not sure if we discussed this, but I plan on taking another look at the chrome inspector backend that I added two weeks ago. At a minimum it will be moved into libs. Hopefully, I can replace it with something much simpler that maintains the same domain/dispatch api